package.json

```diff
@@ -103,7 +103,7 @@
     "shpjs": "^3.5.0",
     "simple-oauth2": "git+https://github.com/bilts/simple-oauth2.git#fix-encoding",
     "source-map-support": "^0.5.19",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "^4.2.0",
```

Comment:
Was there no 5.x version with a fix for the vulnerability? Not sure dropping down a major version is the right thing to do.